source/getting-started/index.markdown

```diff
@@ -49,21 +49,21 @@ We will need a few things to get started with installing Home Assistant. The lat
 
 ### Installing Hass.io
 
-1. Put the SD card in your SD card reader.
+1. Put the SD card in your card reader.
 2. Open balenaEtcher, select the Hass.io image and flash it to the SD card.
-3. Unmount the SD card and remove it from your SD card reader.
-4. Only if you want to configure WiFi or a Static IP (requires USB stick):
-   - Format a USB-Stick to FAT32 with volume-name `CONFIG`.
-   - Create a folder named `network` in the root of the newly formatted USB-stick.
-   - Within that folder create a file named `my-network` without extension.
+3. Unmount the SD card and remove it from your card reader.
+4. Follow this step if you want to configure Wi-Fi or a static IP address (this step requires a flash drive). Otherwise, move to step 5.
```

Comment:
I would not prefer to use a flash drive since that is less specific. An SD card is a flash drive as well.

Reply:
Ah, good point, I'll change it back. I thought USB stick was too general, but given the context in the document, it's obvious what it means.

Any thoughts on referring to the Raspberry Pi 3 B+ vs the Pi 4? Explanation in my original comment above.